docs/DeployingOnCentOS7.md

```diff
@@ -0,0 +1,291 @@
+          "modules": [
+            {
+              "name": "com_mesosphere_mesos_NetworkIsolator",
+              "parameters": [
+                {
+                  "key": "isolator_command",
+                  "value": "/calico/calico_isolator"
+                },
+                {
+                  "key": "ipam_command",
+                  "value": "/calico/calico_isolator"
+                }
+              ]
+            },
+            {
+              "name": "com_mesosphere_mesos_NetworkHook"
+            }
+          ]
+        }
+      ]
+    }
+    EOF
+    $ sudo mv modules.json /calico/
+
+## Run Calico Node
+
+Download `calicoctl`, Calico's command line tool. We will use `calicoctl` to spin up calico-node in a docker container on each slave. 
+
+When doing so, we must point `calicoctl` to our running instance of etcd. Go back and retrieve the IP address of the host where etcd is running. Use that address in the `ECTD_AUTHORITY` variable passed to `calicoctl`.
+
+    $ wget https://github.com/projectcalico/calico-docker/releases/download/v0.9.0/calicoctl
+    $ chmod +x calicoctl
+    $ sudo ETCD_AUTHORITY=<IP of host with etcd>:4001 ./calicoctl node
+
+# Configure & Run Mesos
+
+An explanation of the configuration options for Mesos can be found by running `mesos-init-wrapper -h`. 
+
+We will be need to set the correct environment variables for each slave and master. These environment variables are interpreted as command line arguments in the mesos-slave and mesos-master applications at runtime.
+
+Once these environment variables are set accordingly, we can finally start the Mesos services.
+
+## Mesos Master
+
+### Set Mesos' Master Environment Variables
+
+Append the following lines to `/etc/default/mesos-master` on each of your master hosts. 
+
+The value of `MESOS_QUORUM` may change depending on how many master hosts you have in your cluster. Mesos recommends that the quorum count is at least 1/2 the number of master hosts running. 
+
+Also note, you will need to fill in the IP address of the host that is running ZooKeeper.
+
+    MESOS_QUORUM=1
+    MESOS_WORK_DIR=/var/lib/mesos
+    MESOS_LOG_DIR=/var/log
+    MESOS_ZK=zk://<IP of host with zookeeper running>:2181/mesos
+
+If ZooKeeper is not running on your master, you will need to go into `/etc/mesos/zk` and change the line to include the IP address of the host with ZooKeeper running.
+
+### Run Mesos' Master
+
+Run the mesos-master process on your master host.
+
+    $ sudo systemctl enable mesos-master.service
+    $ sudo systemctl start mesos-master.service
+
+## Mesos Slaves
+
+### Set Mesos' Slaves Environment Variables
+
+Append the following lines to `/etc/default/mesos-slave` on each of your slave hosts. 
+
+Note, you will need to fill in the IP address of the host that is running ZooKeeper and the IP address of the host running etcd.
+
+    MESOS_MASTER=zk://<IP of host with zookeeper running>:2181/mesos
+    MESOS_EXECUTOR_REGISTRATION_TIMEOUT=5mins
```

Comment:
Same with these. I know we copied them from the docker-compose file, but we should weed out whichever ones aren't necessary.